substrate/frame/balances/src/impl_currency.rs

```diff
@@ -100,6 +100,11 @@ mod imbalances {
 			mem::forget(self);
 			(Self(first), Self(second))
 		}
+		fn extract(&mut self, amount: T::Balance) -> Self {
```

Comment:
there is a separate PR for this https://github.com/paritytech/polkadot-sdk/pull/1847